config/routes.rb

```diff
@@ -1,3 +1,4 @@
 Rails.application.routes.draw do
   root "welcome#index"
+  get "/food", to: "foods#index"
```

Comment:
if you do `/foods` instead, that would allow the form to work without the change you made from `form_tag` to `form_with`. the `:foods` in the form_tag line denotes the path that it would direct things too (/foods)